### ISSUE TITLE
SmartString::as_str panic

### ISSUE THREAD

**Issue by JPaja** [TeXitoi/osmpbfreader-rs#44]
SmartString will panic if string length is larger than some value

```rs
    //https://download.geofabrik.de/europe/serbia-latest.osm.pbf
    let path = std::path::Path::new("serbia-latest.osm.pbf");
    let r = std::fs::File::open(&path).unwrap();

    let mut pbf = osmpbfreader::OsmPbfReader::new(r);

    let objs = pbf
        .par_iter()
        .filter(|obj| obj.is_ok())
        .map(|obj| obj.unwrap())
        .filter(|obj|obj.is_relation() && obj.relation().unwrap().tags.contains("name", "Београд"))
        .take(1);
        
    for obj in objs {
        println!("{:?}", obj);
    }
```
outputs
```
thread 'main' panicked at 'assertion failed: len <= MAX_INLINE', /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/smartstring-0.2.10/src/inline.rs:41:9
```
and from stack trace we can see that smartstring::SmartString<Mode>::as_str panics at 
https://github.com/TeXitoi/osmpbfreader-rs/blob/722ccfea8100112b1ec916596a6a72a2380e9a11/src/objects.rs#L32
```
   6: smartstring::SmartString<Mode>::as_str
             at /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/smartstring-0.2.10/src/lib.rs:466:9
   7: osmpbfreader::objects::Tags::contains::{{closure}}
             at /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/osmpbfreader-0.15.2/src/objects.rs:32:43
   8: core::option::Option<T>::map_or
             at /rustc/fc594f15669680fa70d255faec3ca3fb507c3405/library/core/src/option.rs:988:24
   9: osmpbfreader::objects::Tags::contains
             at /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/osmpbfreader-0.15.2/src/objects.rs:32:9
```

**Comment by TeXitoi** [comment]
Looks like a bug in rust or smartstring https://github.com/bodil/smartstring/issues/38

**Comment by TeXitoi** [comment]
fixed by #45
